data/avro-schema/src/test/kotlin/net/corda/data/ledger/persistence/LedgerPersistenceRequestSchemaCompatibilityTest.kt

```diff
@@ -50,6 +50,7 @@ class LedgerPersistenceRequestSchemaCompatibilityTest {
                 "net.corda.data.ledger.persistence.PersistTransactionIfDoesNotExist",
                 "net.corda.data.ledger.persistence.FindTransaction",
                 "net.corda.data.ledger.persistence.FindUnconsumedStatesByType",
+                "net.corda.data.ledger.persistence.FindUnconsumedStatesByExactType",
```

Comment:
I don't think we want changes to this file.

@vlajos I think you said we won't support 5.0 to 5.1 upgrades, therefore we _could_ make these changes in preparation of 5.1 to 5.2 upgrades. But, solely for the changes made in this PR, we don't need these.